stdlib/docs/secp256k1_base_field_math.md

```diff
@@ -0,0 +1,10 @@
+| neg | Given a secp256k1 base field element ( say a ) on stack, represented in Montgomery form<br /><br />( i.e. number having eight 32 -bit limbs ), following function negates it to<br /><br />field element a' \| a' + a = 0<br /><br />Stack expected as below, holding input<br /><br />[a0, a1, a2, a3, a4, a5, a6, a7, ...] \| a[0..8] is a secp256k1 base field element<br /><br />After finishing execution of this function, stack should look like<br /><br />[c0, c1, c2, c3, c4, c5, c6, c7, ...] \| c[0..8] is a secp256k1 base field element<br /><br />See https://github.com/itzmeanjan/secp256k1/blob/6e5e654823a073add7d62b21ed88e9de9bb06869/field/base_field.py#L78-L96 |
+| sub | Given two secp256k1 base field elements, say a, b, ( represented in Montgomery form,<br /><br />each number having eight 32 -bit limbs ) on stack, following function computes modular<br /><br />subtraction of those two operands c = a + (-b) = a - b<br /><br />Stack expected as below, holding input<br /><br />[a0, a1, a2, a3, a4, a5, a6, a7, b0, b1, b2, b3, b4, b5, b6, b7, ...] \| a[0..8], b[0..8] are secp256k1 base field elements<br /><br />After finishing execution of this function, stack should look like<br /><br />[c0, c1, c2, c3, c4, c5, c6, c7, ...] \| c[0..8] is a secp256k1 base field element<br /><br />See https://github.com/itzmeanjan/secp256k1/blob/6e5e654823a073add7d62b21ed88e9de9bb06869/field/base_field.py#L98-L102 |
+| to_mont | Given a 256 -bit number on stack, represented in radix-2^32 form i.e. eight 32 -bit limbs,<br /><br />this routine computes Montgomery representation of provided radix-2^32 number.<br /><br />Stack expected in form<br /><br />[a0, a1, a2, a3, a4, a5, a6, a7, ...]<br /><br />Final stack should look like<br /><br />[a0`, a1`, a2`, a3`, a4`, a5`, a6`, a7`, ...]<br /><br />See section 2.2 of https://eprint.iacr.org/2017/1057.pdf<br /><br />See https://github.com/itzmeanjan/secp256k1/blob/6e5e654823a073add7d62b21ed88e9de9bb06869/field/base_field_utils.py#L225-L232<br /><br />for implementation |
+| from_mont | Given a 256 -bit number on stack, represented in Montgomery form i.e. eight 32 -bit limbs,<br /><br />this routine computes radix-2^32 representation of provided u256 number.<br /><br />Stack expected as<br /><br />[a0, a1, a2, a3, a4, a5, a6, a7, ...]<br /><br />Final stack should look like<br /><br />[a0`, a1`, a2`, a3`, a4`, a5`, a6`, a7`, ...]<br /><br />See section 2.2 of https://eprint.iacr.org/2017/1057.pdf<br /><br />See https://github.com/itzmeanjan/secp256k1/blob/6e5e654823a073add7d62b21ed88e9de9bb06869/field/base_field_utils.py#L235-L241<br /><br />for implementation |
```

Comment:
The backticks cause strange formatting. Would an apostrophe work instead? e.g. a0', a1' instead of a0`, a1`

Reply:
Yes, it's a markdown formatter, didn't notice that. Thanks. Fixed in  0c3f8a84593eb21aa03ba1ab532d3772966a1bf3